### ISSUE TITLE
Swift package manager tag naming convention

### ISSUE THREAD

**Issue by jrief-digikey** [SiftScience/sift-ios#115]
The Xcode Swift package manager seems to prefer tags of the form X.X.X (i.e. 2.1.1).  Can an additional tag be added to the release to support Xcode?

**Comment by viaskal-sift** [comment]
Thanks @jrief-digikey for bringing this, we'll take a look

**Comment by SujishaExalture** [comment]
We cannot add additional tags. The version rule requires Swift packages to conform to semantic versioning . The Xcode Swift package manager  should be tags of the form X.X.X (i.e. 2.1.1). 

A package version is a three period-separated integer, for example 1.0.0. It must conform to the semantic versioning standard in order to ensure that your package behaves in a predictable manner once developers update their package dependency to a newer version. To learn more about the semantic versioning standard, visit https://semver.org

**Comment by jrief-digikey** [comment]
@SujishaExalture Agreed, Xcode is requiring tags following the [semver](https://semver.org/) standards, however Sift is preceding [tags](https://github.com/SiftScience/sift-ios/tags) with a "v".  I'm recommending that Sift drop the preceding "v".

**Comment by SujishaExalture** [comment]
Thanks for the catch. @jrief-digikey . We will consider for sure

**Comment by jordanhbuiltbyhq** [comment]
I had to select the master branch because of this issue, up to next major version doesn't work.

**Comment by andrewjmeier** [comment]
Hey @SujishaExalture can we get an update on this?

**Comment by viaskal-sift** [comment]
@jrief-digikey seems, nowadays this version pattern is already supported by Swift, could you pls confirm the issue is not relevant anymore

**Comment by viaskal-sift** [comment]
@jrief-digikey @andrewjmeier @jordanhbuiltbyhq any objections to closing this issue?
